dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/persistence/dataset/DatasetVersion.java

```diff
@@ -658,6 +659,10 @@ public String getTitle() {
         return retVal;
     }
 
+    public String getParsedTitle() {
+        return Jsoup.parse(getTitle()).text();
```

Comment:
Is our title embeded in html characters that we need to unencode it? Is disabling escape in xhtml not sufficient?